### PR TITLE
VAVSA-37366: Add remaining spec items to billing tpl.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Implements remaining items for TTP billing from https://github.com/department-of-veterans-affairs/va.gov-team/issues/37366
**DOES NOT INCLUDE TBD CERNER WORK**
image

## Testing done
Visual / build

## Screenshots
Facility name in intro text
image

Questions about copay h2 (jump linkable) with system name in text
image

Linked # and facility hours
image

Styled Pay online jump link
image

Linked h4 facility
image

## Acceptance criteria
**Go to `/minneapolis-health-care/billing-and-insurance` and visually verify:**
- [x] Name of facility in intro text
- [x] Presence of h4 `Questions about copay balance` and body text below with facility name
- [x] Phone label and linked # below
- [x] `Pay online, by phone, or by mail` with new arrow style icon
- [x] Pay in person facility wrapped in h4 and linked to facility
- [x] Hours below are correctly displayed

### CODE REVIEW

**Review by maxx1128 — APPROVED** [review]
LGTM 👍

File: src/site/layouts/vamc_system_billing_insurance.drupal.liquid [hunk 1]
```diff
@@ -2,24 +2,46 @@
...
 
+          <div class="usa-content">
+            <h2>Questions about copay balance</h2>
+            <p>
+              For questions about the copay balance of your
+              {{ fieldOffice.entity.entityLabel }}
+              bill, call us toll free at the number below. You won't need to pay
+              any copays for X-rays, lab tests,
+              preventitive tests, and services like health screenings or
+              immunizations.
```
Comment (swirtSJW) [inline src/site/layouts/vamc_system_billing_insurance.drupal.liquid:35]: Why isn't this coming from centralized content?

Comment (swirtSJW) [reply src/site/layouts/vamc_system_billing_insurance.drupal.liquid:35]: Nevermind... I see it.  It is the token replacement thing.

**Review by swirtSJW — APPROVED** [review]
Code works as spec'd.
Approving, but I see it is still in in-progress so maybe I am jumping the gun.

**Pull request merged**